llvm/tools/llvm-cov/CodeCoverage.cpp

```diff
@@ -1270,13 +1270,18 @@ int CodeCoverageTool::doExport(int argc, const char **argv,
                               cl::desc("Don't export branch data (LCOV)"),
                               cl::cat(ExportCategory));
 
+  cl::opt<bool> UnifyInstantiations("unify-instantiations", cl::Optional,
+                                    cl::desc("Unify function instantiations"),
+                                    cl::init(false), cl::cat(ExportCategory));
```

Comment:
Would it be worthwhile to turn this ON by default?  Then the summary would be more consistent with the summary output of `llvm-cov show`.

Reply:
Okay, I changed the default to true.